tests/e2e/tests/content-manager/editview.spec.ts

```diff
@@ -678,5 +678,47 @@ test.describe('Edit View', () => {
       await expect(page.getByRole('tab', { name: 'Draft' })).not.toBeDisabled();
       await expect(page.getByRole('tab', { name: 'Published' })).toBeDisabled();
     });
+
+    test('as a user I want to add a component to a dynamic zone at a specific position', async ({
+      page,
+    }) => {
+      await page.getByLabel('Content Manager').click();
+      await navToHeader(page, ['Content Manager', 'Shop'], 'UK Shop');
+
+      // There should be a dynamic zone with two components
+      const components = await page
+        .getByRole('listitem')
+        .filter({ has: page.getByRole('heading') })
+        .all();
+      expect(components).toHaveLength(2);
+      expect(components[0]).toHaveText(/product carousel/i);
+      expect(components[1]).toHaveText(/content and image/i);
+
+      // Add components add specific locations:
+      // - very last position
+      await components[1].getByRole('button', { name: /more actions/i }).click();
+      await page.getByRole('menuitem', { name: /add component below/i }).dispatchEvent('click');
```

Comment:
I hate that I have to do those dispatchEvent, but a normal click just would not pass in the CI, even though it always passed locally. See [this section](https://playwright.dev/docs/input#forcing-the-click) of the Playwright docs (force: true did not work either in the CI 🤷)